dart/collision/fcl/FCLCollisionNode.h

```diff
@@ -62,12 +63,10 @@ class FCLCollisionNode;
 struct FCLUserData
 {
   FCLCollisionNode* fclCollNode;
-  dynamics::BodyNode* bodyNode;
-  dynamics::Shape* shape;
+  dynamics::ShapeNode* shapeNode;
```

Comment:
Same as for the Bullet user data: I wonder if we should make this a ``WeakNodePtr`` instead. That way if the ShapeNode gets deleted, it can see that and update itself accordingly.